site/pages/what-we-do/header-slice/style.css

```diff
@@ -112,9 +116,15 @@
 
   .arrowUp {
     margin-right: 60px;
+    transform: rotate(-165deg)
+  }
+
+  .arrowDown {
+    transform: rotate(2deg);
```

Comment:
Screenshots would be great to explain this change - should we not just replace the asset if we need to do this?

Reply:
The arrow pointing down has a different inclination on the small screen. 
<img width="279" alt="screen shot 2018-03-09 at 10 37 12" src="https://user-images.githubusercontent.com/822817/37203855-fe624430-2386-11e8-9075-5cfc1b699cd2.png">
<img width="515" alt="screen shot 2018-03-09 at 10 37 01" src="https://user-images.githubusercontent.com/822817/37203857-fe85a11e-2386-11e8-9759-5778cfd2c0c3.png">

And it's the same asset for the arrow pointing up so I went for the transformation.

Reply:
That makes sense 👍

Reply:
Anyway, now that I looked at it, some unnecessary rotations are being done. I'll clean it up.